runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -297,12 +297,15 @@ export class RepeatingFieldPageController extends PageController {
 
     for (let i = 0; i < answers.length; i++) {
       if (typeof answers[i] === "string") {
-        const values = answers[i].split(":");
-        let multiInput = {
-          "type-of-revenue-cost": values[0].trim(),
-          value: values[1].substring(2),
+        // TODO: This is a javascript right split, we need to have a re-think about how add another answers work
+        const reversedAnswer = answers[i].split("").reverse().join("");
+        const values = reversedAnswer.split(" : ");
+        const amount = values.shift().split("").reverse().join("");
+        const description = values.join(" : ").split("").reverse().join("");
+        answers[i] = {
+          "type-of-revenue-cost": description.trim(),
+          value: amount.substring(1),
```

Comment:
It's a bit confusing that the substring happens here, can we do it above when defining `amount` instead? Is this about stripping the pound sign off?

Reply:
Yup, spot on! I moved the substring to the above split for amount